packer/ansible/roles/aggregate/files/aggregate-config.sh

```diff
@@ -49,13 +55,28 @@ showHelp() {
   echo "                     (not recommended if your IP address changes)"
   echo "--http-port  <value> Set a new HTTP port"
   echo "--https-port <value> Set a new HTTPS port"
+  echo "--net-mode   <value> Set the VM's network mode. Use 'nat' or 'bridge'"
 }
 
 if [ ${HELP} = YES ]; then
   showHelp
   exit 0
 fi
 
+if [ ${NET_MODE} != "nat" ] && [ ${NET_MODE} != "bridge" ]; then
+  echo "Error: Invalid --net-mode value. Use 'nat' or 'bridge' according to the VM's networking configuration"
+  echo ""
+  showHelp
+  exit 1
+fi
+
+if [ ${NET_MODE} == "bridge" ] && ([ -z ${HTTP_PORT} ] || [ -z ${HTTPS_PORT} ]); then
```

Comment:
Maybe I'm confused, but this warning is also valid for NAT mode, no?

Reply:
Not really, but that's mainly because there isn't clear criteria of which args are required which aren't.

I'd propose this:
- If `--http-port` is set, then `--https-port` is required, and vice versa.
- If `--http-port` and `--https-ports` are defined, then `--net-mode` is required.

(FQDN is always optional)